documentation/cloud-manager-guide/admin/clusters/cluster-monitoring.md

```diff
@@ -0,0 +1,96 @@
+---
+keywords:
+title: Cluster Monitoring
+description: Cluster monitoring
+---
+# Cluster Monitoring
+
+The status of each cluster and all of its nodes can be monitored from the *Clusters* tab in the *Admin* section of Environment Operations Center. This guide outlines how to review the status and details of a cluster and how to review the status of all nodes within a cluster.
+
+## Getting started
+
+On the *Clusters* tab, each cluster is listed including its region, number of nodes, Kubernetes version, and status.
+
+Depending on the operational health of a cluster its status will be listed as:
+
+> **RL QUESTION: Requested definitions and confirmation for the following statuses**
+
+- Ready:
+- Running:
+- Warning:
+- Operational:
+
+![image description](images/cluster-overview.png)
+
+## View Cluster details
+
+To view further details about a cluster and its nodes, select the cluster name.
+
+![image description](images/select-name.png)
+
+The detailed view of a cluster provides the cluster general info and utilization status. A list of all nodes and their statuses are listed below the cluster details.
+
+![image description](images/cluster-details.png)
+
+### Cluster info
+
+General cluster details are listed under "Cluster Info" including:
+
+> **RL QUESTION: Requested definitions and confirmation for the following statuses**
+
+- Name:
+- Overall status: The current operational status of the cluster. This will either be "Ready", "Running", "Warning", or "Unavailable".
+- Region: The region where cluster the servers are located.
+- Nodes: The number of nodes that make up the cluster.
+- Kubernetes Version: Which version of Kubernetes the cluster is operating on.
+
+![image description](images/cluster-info.png)
+
+### Cluster utilization
+
+Cluster utilization lists the operating status of the cluster, including:
+
+> **RL QUESTION: Requested definitions and confirmation for the following items and status color codes**
+
+- CPU: The amount of CPU used by the cluster.
+- Memory: The amount of memory used by the cluster.
+- Disk: The amount of disk usage by the cluster.
+- Deployments:
+- Jobs:
+
+The status of CPU, Memory, and Disk are color coded to indicate the amount of space used. The color indicators are:
+
+- Green:
+- Yellow:
+- Red:
+
+![image description](images/cluster-utilization.png)
+
+### Node statuses
+
+The "Nodes" section lists all nodes in the cluster and their operating statuses, including their overall status, CPU, Memory, and Disk usage.
```

Comment:
"The "Nodes" section lists all nodes in the cluster, as well as their individual status, CPU, Memory, and Disk usage."